### ISSUE TITLE
Some Mobs Permanently Frozen

### ISSUE THREAD

**Issue by TheMattVid** [Esmorall/EntityTrackerFixer#18]
Hello, I have an issue on my server where some of the mobs have seemingly frozen at random. I believe this occurred sometime around the update to 1.16. The mobs can be pushed, but they don't move, eat, turn their heads, or anything. They are behaving in a similar way to how mobs behave when they are outside of EntityTrackerFixer's range. It's almost as though EntityTrackerFixer has forgotten that these mobs exist and it is no longer checking them anymore to reactivate them. I have tried removing EntityTrackerFixer and all of my other plugins, and the problem does not go away. It is as if these particular mobs are in an 'off' state and are waiting to be turned back on.

Would it possible for EntityTrackerFixer to add a feature where it can look for mobs in the radius of the player that are untracked and turn them back on?

**Comment by GrimEpp** [comment]
Same problem here https://gyazo.com/54787161dd5f84ae99cdb3390a393c13

**Comment by Movinggun** [comment]
Yeah it breaks your mobs completely. I had to write my own plugin that checks for unaware mobs when chunks are loaded and set them to aware to fix this,

**Comment by Elecast2** [comment]
Hi, i will try to fix this next update, but i have some questions ¿does new spawned mob lose intelligency? if the case is not, then try to spawn new entities and go far until they disappear but the chunk is still loaded, wait for etf to untrack process, then put another new mob near you and restart the server, tell me if the far mobs keep or lose their AI, and also what happened to the mobs you summoned near you.

**Comment by Elecast2** [comment]
The problem is when server restart/crash, untracked entities remains unaware if the option disable tick for entities is enabled. I will try to fix this for the next update

**Comment by Elecast2** [comment]
Try this version and tell me if the issue goes away. 
https://drive.google.com/file/d/1H6BP2opL0K1MGpXdne3nn_qdnrdOvkbW/view?usp=sharing

**Comment by Movinggun** [comment]
Your code is almost the exact same to my fix for it so yes that should work.
https://i.imgur.com/CU3WdRO.png

**Comment by djgeorgi** [comment]
Hi no it's not working with this https://drive.google.com/file/d/1H6BP2opL0K1MGpXdne3nn_qdnrdOvkbW/view?usp=sharing

**Comment by josephrooks** [comment]
Also having this problem. Will install the one at your link and see if it makes the mobs in question move again. Will it now just re-scan periodically for mobs that need to be able to move again?

**Comment by josephrooks** [comment]
Just tested the version you released an hour or two ago on the Spigot site, and the problem is fixed for me across everywhere players had reported it. Thanks!

**Comment by ericskh2** [comment]
I'm using Tuinity now which already implements your plugin's feature.
However, when I removed your plugin (latest version) mob will still freeze.
Is there anyway to completely uninstall this plugin?

**Comment by Elecast2** [comment]
Hi, You should keep the plugin installed with "disable-tick-for-untracked-entities" enabled to recover mobs ai, put tps-limit to 0 so the plugin task will not work but only the event that makes frozen entities to recover ai.
Can you tell me which patch of Tuinity implements my plugin's feature?

**Comment by ericskh2** [comment]
The feature is called tuinity optimized-tracker

**Comment by josephrooks** [comment]
Actually came here to ask the same question you just answered - might be a good thing to note in the config file! I'm sure others will notice it if they disable that feature.

**Comment by shoker537** [comment]
@Esmorall I just made a simple plugin to make entity aware in radius near me, mobs become unfrozen. Something is wrong in your unfroze task I guess.

**Comment by spetznack** [comment]
Version 1.3.1 fixes this right? I haven't had this issue on my server after installing that version but I've only had it running for one day so far. 

Server is running Paper version git-Paper-108 (MC: 1.16.1) (Implementing API version 1.16.1-R0.1-SNAPSHOT)

**EDIT:** 
I just had this issue pop up for me as well using 1.3.1: 
![image](https://user-images.githubusercontent.com/1935616/88246093-b6f82600-cc99-11ea-87ad-04d20fd0d94e.png)

Here is my config: 
https://gist.github.com/spetznack/a0afeace86f7c23dd428a81beba5d243

**EDIT2 13.08.2020:** 
This only happened for one day or two and then I didn't see this issue again. It is weird because I had not changed any configurations. This plugin is awesome nevertheless and with this issue gone then it is even more awesome :D

**Comment by TheMattVid** [comment]
Updating this plugin to 1.3.1 had initially unfrozen all the frozen mobs on my server, but after some sort of crash occurred yesterday, a whole bunch of mobs all across the server are once again frozen and this time, the plugin has not unfrozen them.

**Comment by Jerry2001** [comment]
> Yeah it breaks your mobs completely. I had to write my own plugin that checks for unaware mobs when chunks are loaded and set them to aware to fix this,

Do you by any chance could share the plugin. All my mob before 1.16.4 in 1.16.4 are frozen and the plugin hasn't supported for 1.16.4 yet so it is kinda annoying. Thank you.
